dpnp/dpnp_iface_logic.py

```diff
@@ -337,6 +340,207 @@ def any(a, /, axis=None, out=None, keepdims=False, *, where=True):
     return dpnp.get_result_array(usm_res, out)
 
 
+def array_equal(a1, a2, equal_nan=False):
+    """
+    ``True`` if two arrays have the same shape and elements, ``False``
+    otherwise.
+
+    Parameters
+    ----------
+    a1 : {dpnp.ndarray, usm_ndarray, scalar}
+        First input array, expected to have numeric data type.
```

Comment:
`Numeric data type` means any type except boolean. I suppose it isn't true for that function.